utility/application-util.js

```diff
@@ -0,0 +1,31 @@
+const {IN_PROGRESS} = require("../constants/application-constants");
```

Comment:
Let's refrain from developing an application utility as it disrupts the integrity of the Application capsule. We should include outside of application.js

Reply:
The tradeoff with encapsulation is that we lose our ability to unit test these functions. In general we do not want these to be called outside of the application class but they need to be accessible to the unit testing file. I can move these within the application class but then we run into the issue of running static methods within an instantiated class. I am fine with removing this file but I don't think it will solve the issue. I don't know if there is a way keep these scoped to the application capsule while maintaining the unit testing capability

Reply:
I've moved these functions to application.js to preserve encapsulation for now and removed the unit tests. The unit tests in this case were very minor but we will need to either find a way to unit test private methods in the future or expose them to the test files